src/styles/Paragraph.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import {size, fontSize} from "./variables";
+
+const Container = styled.p`
```

Comment:
this should be "Paragraph" or "ParagraphContainer" IMO

Reply:
I don't agree but I only care a little ;-) If the maintainers want it changed, I'll do it.